CPP11.md

```diff
@@ -72,7 +72,18 @@ Type deduction with lvalues and rvalues:
 int x = 0; // `x` is an lvalue of type `int`
 int& xl = x; // `xl` is an lvalue of type `int&`
 int&& xr = x; // compiler error -- `x` is an lvalue
-int&& xr2 = 0; // `xr2` is an lvalue of type `int&&` -- binds to the rvalue temporary, `0`
+int&& xr2 = 0; // `xr2` is an rvalue of type `int&&` -- binds to the rvalue temporary, `0`
```

Comment:
This comment change isn't correct. `xr2` is still an lvalue, but its type is `int&&`.